website/src/components/FontPreview.tsx

```diff
@@ -60,13 +61,19 @@ export const FontPreview = ({ defPreviewText, metadata }: FontPreviewProps) => {
     });
   }, [events, defPreviewText, defStyle, defWeight]);
 
-  const downloadLink = fontsourceDownload.cssDownload(
-    metadata.fontId,
-    weight,
-    style
-  );
+  const [fontLoaded, setFontLoaded] = useState(false);
+
+  useEffect(() => {
+    setFontLoaded(false);
+
+    // Give browser time to unload fonts in order to not cause a flash of the "Fallback Outline" font
+    const timeout = setTimeout(
+      () => document.fonts.ready.then(() => setFontLoaded(true)),
+      500
```

Comment:
Just out of curiosity, do you know why we have the flash of Fallback Outline font in the first place? I'm a little wary of a fixed timeout value, although it does make the experience more consistent and smoother onsite.

Reply:
Not really, but I'm guessing it's because it hasn't been put into the system yet that fonts need to load, so when we call `document.fonts.ready` it instantly sets the `fontLoaded` state to true. In giving it the half-second, it allows the system to realize that there actually are fonts that need to be loaded.